package/src/components/CheckoutAction/v1/CheckoutAction.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+class CheckoutAction extends Component {
+  static propTypes = {
+    /**
+     * The component to display if workflow status is `active`
+     */
+    activeStepElement: PropTypes.node.isRequired,
+    /**
+     * The component to display if workflow status is `complete`
+     */
+    completeStepElement: PropTypes.node.isRequired,
+    /**
+     * The component to display if workflow status is `incomplete`
+     */
+    incompleteStepElement: PropTypes.node.isRequired,
+    /**
+     * Label of workflow step
+     */
+    label: PropTypes.string.isRequired,
+    /**
+     * Status of current checkout step
+     */
+    status: PropTypes.string.isRequired,
```

Comment:
Limit the status values by using `status: PropTypes.oneOf(['active', 'inactive', 'completed .... etc ]).isRequired,
`